js/src/views/Signer/components/TransactionPending/TransactionPending.js

```diff
@@ -65,20 +65,24 @@ export default class TransactionPending extends Component {
   render () {
     const { totalValue } = this.state;
     const className = this.props.className || '';
+
+    const { gasPriceEthmDisplay, gasToDisplay } = this.state;
+    const { id, date, data } = this.props;
+
     return (
       <div className={ `${styles.container} ${className}` }>
         <div className={ styles.mainContainer }>
           <TransactionMainDetails
             { ...this.props }
             className={ styles.transactionDetails }
             totalValue={ totalValue }>
-            <div className={ styles.iconsContainer }>
-              { this.renderGasPrice() }
-              { this.renderData() }
-            </div>
-            <div className={ styles.expandedContainer }>
-              { this.renderDataExpanded() }
-            </div>
+            <TransactionSecondaryDetails
```

Comment:
As it stands, for the Signer is is probably the best approach. Everything is very, very, very modular and re-used among the different Pending/Completed layouts. Best way of fitting in with the style that is there.